retroshare.pri

```diff
@@ -501,8 +501,16 @@ no_rs_deprecatedwarning {
 }
 
 no_rs_cppwarning {
+    QMAKE_CXXFLAGS += -Wno-unknown-warning-option
```

Comment:
Ok with this, while the others are concerning, I'd prefer we fix those other warnings and personally i take of them in the code i edit

Reply:
Me too be some not...
And now with QtCreator you have direct warning with clang while editing.
 This is only to show "important" warning not to lose in the crowd...
And we still can use "CONFIG+=rs_cppwarning".